test/DGmethods/integral_test_sphere.jl

```diff
@@ -50,22 +50,18 @@ function init_aux!(m::IntegralTestSphereModel, aux::Vars, g::LocalGeometry)
 
   x,y,z = g.coord
   aux.r = hypot(x, y, z)
-  aux.θ = atan(y , x)
-  aux.ϕ = asin(z / aux.r)
-
+  θ = atan(y, x)
+  ϕ = asin(z / aux.r)
   # Exact integral
-  a = 1 + sin(aux.θ)^2 + sin(aux.ϕ)^2
-  aux.int.v = exp(-a * aux.r^2) - exp(-a * m.Rinner^2)
-  aux.rev_int.v = exp(-a * m.Router^2) - exp(-a * aux.r^2)
+  aux.a = 1 + cos(ϕ)^2 * sin(θ)^2 + sin(ϕ)^2
```

Comment:
I had to reformulate this to make it more stable near the poles (`sin(θ)` can fluctuate considerably).